src/gui/guiFormSpecMenu.h

```diff
@@ -403,6 +350,13 @@ class GUIFormSpecMenu : public GUIModalMenu
 		GUITable::TableOptions table_options;
 		GUITable::TableColumns table_columns;
 
+		struct {
+			bool slotborder = false;
+			video::SColor slotbg_n = video::SColor(255, 128, 128, 128);
+			video::SColor slotbg_h = video::SColor(255, 192, 192, 192);
+			video::SColor slotbordercolor = video::SColor(200, 0, 0, 0);
+		} inventorylist_options;
```

Comment:
Consider moving this to `guiInventoryList.h` so that you can pass/use the entire struct in those two places;

https://github.com/minetest/minetest/pull/9287/files#r363431743
https://github.com/minetest/minetest/pull/9287/files#r363431900

Reply:
Done.